src/main/modules/oidc/index.ts

```diff
@@ -68,13 +60,21 @@ export class OidcMiddleware {
 
     app.get(LOGOUT_URL, async (req: Request, res: Response) => {
       if (req.session.user) {
-        await Axios
-          .get(endSessionUrl)
-          .catch(error => {
-            this.logger.error(error);
-            return error;
-          });
-        req.session.destroy(() => res.redirect(LOGIN_URL));
+        try {
+          await Axios.get(
+            IDAM_API + endSession,
```

Comment:
Are we not able to use the IDAM web public URL here for end session? Thought it will route to API to keep it consistent with the other sign in URLs?

Reply:
@KianKwa - the request is blocked by Azure WAF, probably due to not being configured to accept 'id_token_hint' and 'post_logout_redirect_uri' query params.